src/main/java/mods/railcraft/common/util/misc/MiscTools.java

```diff
@@ -50,10 +50,6 @@ public static void registerTrack(EnumTrack rail) {
             }
     }
 
-    public static String cleanTag(String tag) {
-        return tag.replaceAll("[Rr]ailcraft\\p{Punct}", "").replaceFirst("^tile\\.", "").replaceFirst("^item\\.", "");
```

Comment:
All of my registry changes were pretty much done as the game would immediately crash when you attempted to register an item twice. I think this is stuff that I don't understand properly so I won't change any block or item registration stuff in future and leave it to you to sort out.

> On 30 Mar 2016, at 09:32, CovertJaguar <notifications@github.com> wrote:
> 
> In src/main/java/mods/railcraft/common/util/misc/MiscTools.java:
> 
> > @@ -50,10 +50,6 @@ public static void registerTrack(EnumTrack rail) {
> >              }
> >      }
> >  
> > -    public static String cleanTag(String tag) {
> > -        return tag.replaceAll("[Rr]ailcraft\\p{Punct}", "").replaceFirst("^tile\\.", "").replaceFirst("^item\\.", "");
> Ok...now I'm confused. I'm pretty sure that removing this will break pretty much everything.
> 
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
>